dropwizard-jersey/pom.xml

```diff
@@ -113,5 +113,10 @@
             <groupId>javax.activation</groupId>
             <artifactId>javax.activation-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.bytebuddy</groupId>
+            <artifactId>byte-buddy</artifactId>
+            <version>1.8.13</version>
```

Comment:
this should probably be defined up in `dropwizard-bom`

Reply:
updated